doc/nrf/libraries/caf/power_manager.rst

```diff
@@ -47,139 +47,140 @@ By default, the timeout is set to 120 seconds.
 
 The :kconfig:option:`CONFIG_CAF_POWER_MANAGER_ERROR_TIMEOUT` option sets the period of time after which the device is turned off upon an internal error.
 
-Optional boolean for keeping the system on
-==========================================
+Blocking system off
+===================
 
-The :kconfig:option:`CONFIG_CAF_POWER_MANAGER_STAY_ON` Kconfig option lets the system stay on also when there are no active connections.
+The :kconfig:option:`CONFIG_CAF_POWER_MANAGER_STAY_ON` Kconfig option prevents system off (:c:func:`sys_poweroff`) even if there are no active :ref:`power state restrictions <caf_power_manager_power_state_restrictions>`.
+If the option is enabled, device stays in suspended state, but will not enter the system off state.
+If power off functionality (:kconfig:option:`CONFIG_POWEROFF`) is disabled, the module never enters the system off state (:kconfig:option:`CONFIG_CAF_POWER_MANAGER_STAY_ON` is enabled and has no prompt - cannot be changed by the user).
 
-For more information about configuration options, check the help in the configuration tool.
+Forcing power down
+==================
+
+If the :c:struct:`force_power_down_event` is enabled in configuration (:kconfig:option:`CONFIG_CAF_FORCE_POWER_DOWN_EVENTS`), any application module can submit the event to force a quick power down without waiting.
+The event triggers instantly suspending the application.
+
+Clearing reset reason
+=====================
+
+The module by default clears the reset reason register (``RESETREAS``) right before entering the system off state (right before calling :c:func:`sys_poweroff`).
+This is done to avoid starting serial recovery if nobody has cleared it already.
+Disable the :kconfig:option:`CONFIG_CAF_POWER_MANAGER_CLEAR_RESET_REASON` Kconfig option to prevent this behavior.
+Clearing reset reason functionality is not supported for nRF54H Series SoC (:kconfig:option:`CONFIG_SOC_SERIES_NRF54HX`).
+
+.. note::
+  The reset reason register is not cleared in case system is turned off after a fatal error.
 
 Implementation details
 **********************
 
 The |power_manager| is started when the "main" is ready (which is reported using :c:struct:`module_state_event`).
+The module is responsible for controlling application power state.
+A dedicated set of application events can be used to communicate with the |power_manager|.
 
-When started, it can perform the following operations:
-
-* Manage `Power states`_
-* Trigger `Switching to low power`_
-* Handle `Wake-up scenarios`_ from suspended or off states
-
-Power states
-============
+Application power state
+=======================
 
 The application can be in the following power states:
 
 * `Idle`_
-* `Suspended`_
-* `Off`_
-* `Error`_
-
-The |power_manager| takes control of the overall operating system power state.
+* `Suspended/Off`_
```

Comment:
Why merging those two states?

Reply:
off state is actually suspended state with `sys_poweroff`. As we always go through suspended before going to off, grouping these states and separating them from error states might improve readability.

Reply:
I think it is better to keep both state separately. Suspended/off gives an impression that its either Suspended or off state, which is not the case.